### DISCUSSION TITLE
Apply light and dark theme

### DISCUSSION THREAD

**Discussion by vanillajonathan** [GradienceTeam/Gradience#673] [Ideas]
Some themes have both light and dark variants (example Gruvbox).
I would like to apply both the light and the dark variant of the theme so that it switches between the two when I click on the "Dark Mode" toggle in GNOME

**Comment by daudix** [comment]
Sorry, but Libadwaita does not support light/dark color schemes, like, it impossible, but we will implement scheme switching, but it will require app re-launch

**Reply by daudix** [reply]
This is tracked in #12

**Reply by vanillajonathan** [reply]
Hmm, maybe this could be solved without app relaunch if you filed an issue or  a pull request upstream in libadwaita?

It would be preferable to not have the app re-launch.

**Reply by daudix** [reply]
Libadwaita devs will immediately say go f yourself as they don't support themes and against it

**Reply by daudix** [reply]
Some stuff about libadwaita theming, nothing helps with light/dark themes

https://blogs.gnome.org/alatiera/2021/09/18/the-truth-they-are-not-telling-you-about-themes

https://web.archive.org/web/20210902014740/https://people.gnome.org/~federico/blog/gnome-themes.html

https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/named-colors.html

**Reply by HubKing** [reply]
Why are the LibAdw deves so thoroughly against any customisation? I don't even want crazy different looks like making Gnome look like Windows XP or macOS. All I want is some colour customisation, like making the background darker and text whiter in the dark theme. What's the harm of changing those colours? I don't want to change those because it shows my personality or that sort of crap; I need it for practical reasons, because the low-contrast of the default dark theme makes it difficult for me to use the PC, and I am sure that I am not the only person who feels that.

**Reply by osomfinch** [reply]
Omg! Such a shame. It would be amazing if I could decide on color scheme for both light and dark themes.
